app/templates/components/event-card.hbs

```diff
@@ -23,8 +23,18 @@
           {{moment-format event.startTime 'ddd, MMM DD HH:mm A'}}
         </span>
       </div>
-      <div class="description">
```

Comment:
if the final `else` clause executes, the class 'description' is lost completely.
Also, please tell why the extra class 'location' is added, what does it do?